t/01-test_needle.t

```diff
@@ -168,7 +166,7 @@ needle_init;
 
 my @alltags = sort keys %needle::tags;
 my @needles = @{needle::tags('none') || []};
-is(@needles, 4, "four needles found");
+is(@needles, 21, "21 needles found");
```

Comment:
Where are 21 needles come from now?

Reply:
The 21 needles are different test cases. The OCR test requires additional ones, for which reason they sum up to 21 instead of the previous four.